app/modules/transcribe/annotations/annotations.factory.js

```diff
@@ -57,8 +60,36 @@ function AnnotationsFactory(localStorageService) {
         return annotation;
     }
 
+
+    function checkVariants(annotation) {
+        //splits string by whitespace (different encodings)
+        var lemmas = annotation.text.split(/\s|&nbsp;/g);
```

Comment:
`\s` match all white spaces and `&nbsp;` is added when sw-tags are added. So I need them to have the words separated correctly. Good call on commas, I'll add them in.